data/transactions/logic/pairing.go

```diff
@@ -477,20 +477,18 @@ func bls12381G1MultiMulLarge(points []bls12381.G1Affine, scalarBytes []byte) ([]
 func bls12381G1MultiMulSmall(points []bls12381.G1Affine, scalarBytes []byte) ([]byte, error) {
 	// There must be at least one point. Start with it, rather than the identity.
 	k := new(big.Int).SetBytes(scalarBytes[:scalarSize])
-	var sum bls12381.G1Jac
-	sum.ScalarMultiplicationAffine(&points[0], k)
+	var sum bls12381.G1Affine
+	sum.ScalarMultiplication(&points[0], k)
```

Comment:
I do not understand this change

Reply:
Elliptic curve points are represented in affine coordinates and for certain operations like scalar multiplication the points are first converted to jacobian representation to perform the operation more efficiently and then back to affine.
The old code was explicitly doing the conversion back and forth, the new code leverages a new api that performs the conversion for you.
From v0.13.0 gnark-crypto removed `ScalarMultiplicationAffine`, so the old code would break

Reply:
thank you for clarification!